android/src/main/java/com/imagepicker/ImagePickerModule.java

```diff
@@ -579,6 +579,25 @@ private String getBase64StringFromFile(String absoluteFilePath) {
     return Base64.encodeToString(bytes, Base64.NO_WRAP);
   }
 
+    /**
+   * If necessary, set a sample size > 1 to request the decoder to subsample the
+   * original image, returning a smaller image to save memory.
+   */
+  private int getSampleSize(final String realPath) {
+    File file = new File(realPath);
+    int maxMB = 5;
+    long maxByte = maxMB * 1024 * 1024;
+    long length = file.length();
+    int sampleSize = 1;
+
+    while (length > maxByte) {
+      length /= 2;
+      sampleSize *= 2;
```

Comment:
The sample size goes both vertically and horizontally. So the previous line should be `length /= 4`, right?